api/lib/domain/read-models/ComplementaryCertificationCourseResultForJuryCertificationWithExternal.js

```diff
@@ -33,6 +33,7 @@ class ComplementaryCertificationCourseResultForJuryCertificationWithExternal {
       level: externalLevel,
     });
     this.allowedExternalLevels = allowedExternalLevels;
+    this.defaultJuryOptions = Object.values(ComplementaryCertificationCourseResult.juryOptions);
```

Comment:
On ne peut pas plutôt faire ça puisque qu'on exporte direct les juryOptions ? 

```suggestion
    this.defaultJuryOptions = Object.values(juryOptions);
```

Reply:
On peut 👍 
Il n'y a pas forcément d'ambigüité sur d'où ça vient